pm2.config.js

```diff
@@ -1,5 +1,20 @@
 module.exports = {
   apps: [
+    {
+      name: 'Chronos',
+      script: 'dist/chronos.js',
+      instances: 1, // there can only be 1
```

Comment:
@igorlesnenko today, this is what's making sure we only have 1 chronos running at a time. 
in the future, like @rafaelromcar-parabol says, we'll ditch pm2 for k8s & only deploy a single pod